packages/ui/src/types/storage/file-uploader/types.ts

```diff
@@ -0,0 +1 @@
+export type FileName = string | Array<string>;
```

Comment:
I was following the same pattern we have in the Authenticator where types that are used in the UI package are in their own file in the types folder. This will/could expand to more types as I add more utilities.

Reply:
Prefer we add those files once we need them but won't block on it.

Something i missed here is that we should break this in to two types:
```
type FileName = string;
type FileNames = string[];
```

Although, since these are just primitive values, maybe we don't need the the types at all? For example, we could just have:
```
fileNames: string[];
```
in place of:
```
fileNames: FileNames;
```
This might make it simpler for readers in the future, what do you think?

Reply:
I do like this, so that customers don't have to cmd+click into our types